package.json

```diff
@@ -84,7 +84,7 @@
     "semver": "^5.5.0",
     "seq": "0.3.5",
     "tap": "^10.7.2",
-    "temp": "^0.8.1",
+    "temp": "0.8.3",
```

Comment:
```suggestion
    "temp": "~0.8.3",
```
?

Reply:
the breakage here is in `0.8.4`

Reply:
edited to show the intent with = then :-)